rpcs3/main.cpp

```diff
@@ -451,7 +451,7 @@ int main(int argc, char** argv)
 		}
 
 		// Limit log size to ~25% of free space
-		log_file = logs::make_file_listener(fs::get_cache_dir() + "RPCS3.log", stats.avail_free / 4);
+		log_file = logs::make_file_listener(fs::get_cache_dir() + "Log", stats.avail_free / 4);
```

Comment:
I hardly see a point of renaming it anymore.

Reply:
+1 Users already know of rpcs3.log as the filename.
We gain nothing by renaming it, but lose because it requires end users to learn something new, making it purely detrimental.